src/scenes/Organizations/Create/CreateOrganizationForm.tsx

```diff
@@ -7,25 +13,44 @@ import { SubmitButton, SubmitError, TextField } from '../../../components/form';
 export type CreateOrganizationFormProps = Pick<
   FormProps<CreateOrganizationInput>,
   'onSubmit' | 'initialValues'
-> & { className?: string };
+> & {
+  className?: string;
+  open: boolean;
+  onClose: (open: boolean) => void;
+  close?: boolean;
+};
 
 export const CreateOrganizationForm = ({
   className,
+  open,
+  onClose,
+  close: closeProp,
   ...props
-}: CreateOrganizationFormProps) => (
-  <Form {...props}>
-    {({ handleSubmit }) => (
-      <Card component="form" onSubmit={handleSubmit} className={className}>
-        <CardContent>
-          <SubmitError />
-          <TextField
-            name="organization.name"
-            label="Name"
-            placeholder="Enter organization name"
-          />
-          <SubmitButton />
-        </CardContent>
-      </Card>
-    )}
-  </Form>
-);
+}: CreateOrganizationFormProps) => {
+  const close = closeProp ? closeProp : false;
+  return (
+    <Dialog open={open} onClose={() => onClose(close)} className={className}>
+      <DialogTitle>Create Organization</DialogTitle>
+      <DialogContent>
+        <Form {...props}>
+          {({ handleSubmit }) => (
+            <form onSubmit={handleSubmit}>
+              <SubmitError />
+              <TextField
+                name="organization.name"
+                label="Name"
+                placeholder="Enter organization name"
+              />
+              <SubmitButton />
```

Comment:
`SubmitButton` should be next to the cancel button. You'll need to move the `Form` component higher so `SubmitButton` can work inside of `DialogActions`